### PR TITLE
Add MyDNS.JP suffixes

### PR DESCRIPTION
**Pull request by disco-v8**
Hello!

My Service, https://www.mydns.jp is a Dynamic DNS Service in JAPAN.
I providing subdomains for users.
Please add these domains to PSL. :-)

Regards.

### CODE REVIEW

**Comment by mattn** [comment]
@disco-v8 Thanks your working on this. Seems to be some problems in this change.

https://community.letsencrypt.org/t/limitation-of-jp-tld-suffix/53063/7?u=mattn

* The : on the first line has no spaces around it.
* The email is not in the proper format (at instead of @).
* The email line does not start with Submitted by.

**Comment by disco-v8** [comment]
Hello! mattn.

OK, I'm try again.  :-)

**Comment by mattn** [comment]
@weppos please take a look

```
$ dig TXT _psl.mydns.jp | grep public
_psl.mydns.jp.          299     IN      TXT     "https://github.com/publicsuffix/list/pull/610"
```

**Comment by mattn** [comment]
@disco-v8 Please look this https://github.com/publicsuffix/list/issues/638#issuecomment-380955494 cc: @weppos

This change seems not comply with guidelines.

Thanks.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12389,4 +12389,21 @@ za.org
 // Submitted by Olli Vanhoja <olli@zeit.co>
 now.sh
 
+// MyDNS.JP ： https://www.mydns.jp
+// T.Kabu <kabu@fvg-on.net>
```
Comment (mattn) [inline public_suffix_list.dat]: Must be:

```
// Submitted by T.Kabu <kabu@fvg-on.net> 
```

Comment (disco-v8) [reply public_suffix_list.dat]: OK...
Hummm... I think so it is no good to write a real email address. :-(

File: public_suffix_list.dat [hunk 2]
```diff
@@ -12389,4 +12389,21 @@ za.org
 // Submitted by Olli Vanhoja <olli@zeit.co>
 now.sh
 
+// MyDNS.JP ： https://www.mydns.jp
+// Submitted by T.Kabu <kabu@fvg-on.net>
```
Comment (mattn) [inline public_suffix_list.dat]: please remove trailing whitespace.

File: public_suffix_list.dat [hunk 3]
```diff
@@ -12389,4 +12389,21 @@ za.org
 // Submitted by Olli Vanhoja <olli@zeit.co>
 now.sh
 
+// MyDNS.JP ： https://www.mydns.jp
```
Comment (mattn) [inline public_suffix_list.dat]: @disco-v8 `：` is 0xff1a. please use 0x3a ':' instead.

**Comment by weppos** [comment]
No follow up.

**Pull request closed without merging**